### ISSUE TITLE
Flow test errors

### ISSUE THREAD

**Issue by hzoo** [babel/babel-eslint#103]
I ran all the acorn flow tests in https://github.com/babel/babel/blob/master/test/acorn/tests-flow.js and got 3 failures - saying "unexpected token". Not sure how to go about fixing these?

```js
it("flow 66", function () {
      verifyAndAssertMessages(
        'var identity: <T>(x: T) => T', {}, []
      );
    });
it("flow 67", function () {
      verifyAndAssertMessages(
        'var identity: <T>(x: T, ...y:T[]) => T', {}, []
      );
    });
it("flow 103", function () {
      verifyAndAssertMessages(
        'var a : { <T>(x: T): number; }', {}, []
      );
    });
```

**Comment by sebmck** [comment]
You're on an old version of Babel.

**Comment by hzoo** [comment]
It looks like I'm using `5.4.3` - I removed `node_modules, npm cache clean, npm install` and got the same issue.

**Comment by sebmck** [comment]
Do you have a stacktrace?

**Comment by hzoo** [comment]
I'm not getting anything useful.

```sh
1) verify flow flow 66:
     Error: Expected 0 message(s), got 1 [{"fatal":true,"severity":2,"message":"Unexpected token","line":1,"column":14}]
      at verifyAndAssertMessages (test/non-regression.js:18:11)
      at Context.<anonymous> (test/non-regression.js:585:9)
```
```js

  if (messages.length !== expectedMessages.length) {
    throw new Error("Expected " + expectedMessages.length + " message(s), got " + messages.length + " " + JSON.stringify(messages)); // line 18
  }

//
      it("flow 66", function () {
        verifyAndAssertMessages( // line 585
          'var identity: <T>(x: T) => T', {}, []
        );
      });
```

**Comment by hzoo** [comment]
Ok thanks!
